src/components/FloatingLabel/FloatingLabel.tsx

```diff
@@ -7,9 +7,9 @@ import type { DeepPartial } from '../../types';
 import type { FlowbiteSizes } from '../Flowbite';
 
 export interface FlowbiteFloatingLabelTheme {
-  input: any;
-  label: any;
-  helperText: any;
+  input: any; // eslint-disable-line
```

Comment:
I think that for both: `input` and `label` it would be necessary to create a type.